internal/backends/python/python.go

```diff
@@ -776,13 +777,25 @@ func makePythonUvBackend() api.LanguageBackend {
 			defer span.Finish()
 			// Initalize the specfile if it doesnt exist
 			if !util.Exists("pyproject.toml") {
-				cmd := []string{"uv", "init", "--no-progress"}
+				// uv (currently?) creates a "hello.py" on uv init. Ensure it gets deleted before control returns to the user.
+				sampleFileName := "hello.py"
+				if _, err := os.Stat(sampleFileName); errors.Is(err, os.ErrNotExist) {
+					// Turns out it didn't exist after all!
+					sampleFileName = ""
+				}
+
+				cmd := []string{"uv", "init", "--no-progress", "--no-readme", "--no-pin-python"}
 
 				if projectName != "" {
 					cmd = append(cmd, "--name", projectName)
 				}
 
 				util.RunCmd(cmd)
+				if sampleFileName != "" {
+					if _, err := os.Stat(sampleFileName); errors.Is(err, os.ErrNotExist) {
+						os.Remove(sampleFileName)
+					}
```

Comment:
Seems like the logic is inverted, right now we only delete the file if the file doesn't exit?

Reply:
Yep, addressed